File: runtimes/php/bin/emit-rust-client-fixtures.php
#!/usr/bin/env php
<?php

declare(strict_types=1);

require dirname(__DIR__) . '/vendor/autoload.php';

use Twilic\InteropFixtures;

try {
    fwrite(STDOUT, InteropFixtures::emitInteropFixtures());
} catch (Throwable $err) {
    fwrite(STDERR, 'emit fixtures: ' . $err->getMessage() . PHP_EOL);
    exit(1);
}

File: runtimes/dart/bin/emit_rust_client_fixtures.dart
import 'dart:io';

import 'package:twilic/twilic.dart';

void main() {
  try {
    // ignore: avoid_print
    print(emitInteropFixtures());
  } catch (e, st) {
    // ignore: avoid_print
    print('emit fixtures: $e\n$st');
    // ignore: avoid_exit
    exitCode = 1;
  }
}
